### ISSUE TITLE
Hitting indicaor first time table an graph shows data start value 8 times

### ISSUE THREAD

**Issue by christoph-LSN** [open-sdg/open-sdg#1806]
@brockfanning @MoBosse 

Site repository URL: [https://github.com/christoph-LSN/MT_Site/](https://github.com/christoph-LSN/MT_Site/)]
Open SDG version: [2.1.0-dev] (also tested in 2.0.0)
[2.1.5 data](https://github.com/christoph-LSN/IM_Data/blob/develop/data/indicator_2-1-5.csv)
[2.1.5 meta](https://github.com/christoph-LSN/IM_Data/blob/develop/meta/2-1-5.md)
[data repo](https://github.com/christoph-LSN/IM_Data/)

**Describe the bug**
hitting Indicator [2.1.5](https://christoph-lsn.github.io/MT_Site/2-1-5/) the first time shows Niedersachsen, insgesamt, Duldung 8 times! (table and graph) 
Changing the unit from Prozent to Anzahl it shows again 8 times
After selecting any other disaggregation it works perfektly. Only watching the indicator the first time this error occures.

in the [2-1-5-indicator.csv](https://github.com/christoph-LSN/IM_Data/blob/develop/data/indicator_2-1-5.csv)  2021,"Niedersachsen","insgesamt","Duldung" exist only two times for Anzahl in line 1951 and for Prozent in line 9151 

![image](https://user-images.githubusercontent.com/94443931/188904248-154bb44b-7d41-4ca8-9916-ea20578e134d.png)

![image](https://user-images.githubusercontent.com/94443931/188904342-d2471c16-fdef-4703-a8b7-657a339bccb1.png)

**Comment by christoph-LSN** [comment]
Using only data_start_values without the Unit solves the problem.

But now I have the problem, that one disaggregation "Anteil der Ausländerinnen und Ausländer ohne Aufenthaltstitel" only exists with the unit Prozent. There are noch values for Anzahl. Therefore hitting Anzahl if you have selectet "Anteil der Ausländerinnen und Ausländer..." runs into "this data is not avaliable" (this is ok to me) But if you are in unit Anzahl and then selecting "Anteilder Ausländerinnen und Ausländer..." nothing will happen.

**Comment by brockfanning** [comment]
@christoph-LSN Can you post the "data_start_values" configuration that causes the problem?

**Comment by christoph-LSN** [comment]
@brockfanning I was following the advice of @LucyGwilliamAdmin in this discussion: https://github.com/open-sdg/open-sdg/discussions/1453

using this:

```
data_start_values:
  - unit: Anzahl
    field: Gebietseinheit
    value: Niedersachsen
    field: Geschlecht
    value: insgesamt
    field: ohne Aufenthaltstitel
    value: Duldung
  - unit: Prozent
    field: Gebietseinheit
    value: Niedersachsen
    field: Geschlecht
    value: insgesamt
    field: ohne Aufenthaltstitel
    value: Anteil der Ausländerinnen und Ausländer ohne Aufenthaltstitel

````
produces the following error: 
`ruamel.yaml.constructor.DuplicateKeyError: while constructing a mapping
  in "meta/2-1-5.md", line 28, column 5
found duplicate key "field" with value "Geschlecht" (original value: "Gebietseinheit")
  in "meta/2-1-5.md", line 31, column 5`

using this configuration produces no error, but showing the named problem

[2-1-5.md](https://github.com/christoph-LSN/IM_Data/edit/develop/meta/2-1-5.md) Line 27
[Indicator 2-1-5](https://christoph-lsn.github.io/MT_Site/2-1-5/)

```
`data_start_values:
  - unit: Anzahl
  - field: Gebietseinheit
    value: Niedersachsen
  - field: Geschlecht
    value: insgesamt
  - field: ohne Aufenthaltstitel
    value: Duldung
  - unit: Prozent
  - field: Gebietseinheit
    value: Niedersachsen
  - field: Geschlecht
    value: insgesamt
  - field: ohne Aufenthaltstitel
    value: Anteil der Ausländerinnen und Ausländer ohne Aufenthaltstitel`
```

**Comment by MoBosse** [comment]
@christoph-LSN could you try this please:

>data_start_values:
> - unit: Anzahl
>    field: Gebietseinheit
>    value: Niedersachsen
> - unit: Anzahl
>    field: Geschlecht
>    value: insgesamt
> -  unit: Anzahl
>    field: ohne Aufenthaltstitel
>    value: Duldung
>  - unit: Prozent
>    field: Gebietseinheit
>    value: Niedersachsen
>  - unit: Prozent
>    field: Geschlecht
 >   value: insgesamt
>  - unit: Prozent
>    field: ohne Aufenthaltstitel
>    value: Anteil der Ausländerinnen und Ausländer ohne Aufenthaltstitel

**Comment by christoph-LSN** [comment]
@MoBosse I have tried it, but the result is the same....

**Comment by christoph-LSN** [comment]
@brockfanning any ideas for this problem?

**Comment by brockfanning** [comment]
@christoph-LSN Can you describe the problem again? Is it the original problem with the 8 times, or the "duplicate key" error, or something else?

**Comment by christoph-LSN** [comment]
@brockfanning it is the original problem with the 8 times.

![image](https://user-images.githubusercontent.com/94443931/195769498-b208a11b-4576-4730-a19e-f7e1418a12c7.png)

I only appears looking at the indicator for the first time. Hitting some other disaggregations e.g. Geschlecht for only one time chart and table are displying everthing normally.

![image](https://user-images.githubusercontent.com/94443931/195769879-b48d5276-09ca-4c38-bac5-24b29cfd04e5.png)

then turing off "männlich" and everything is right. I think it has somethinng to do with the data_start_value for units.

![image](https://user-images.githubusercontent.com/94443931/195770002-5e529fb2-b43d-41a2-a505-377b9e15096a.png)

**Comment by brockfanning** [comment]
@christoph-LSN If possible you try out this pull-request and see if it fixes this issue? #1864

**Comment by christoph-LSN** [comment]
@brockfanning looks good!

**Comment by brockfanning** [comment]
This is fixed in 2.1.0 so I'll close this.
